EncoreTickets.SDK.Tests/UnitTests/Api/Results/ApiResultTests.cs

```diff
@@ -186,46 +186,47 @@ public void GetDataOrContextException_IfContextWithoutInfosAsErrors_ReturnsData(
             Assert.AreEqual(data, actual);
         }
 
-        [TestCaseSource(typeof(ApiResultTestsSource), nameof(ApiResultTestsSource.GetDataOrContextException_IfContextWithInfosAsErrors_ThrowsException))]
+        [TestCaseSource(typeof(ApiResultTestsSource), nameof(ApiResultTestsSource.DataOrContextException_IfContextWithInfosAsErrors_ThrowsException))]
```

Comment:
Probably, we need to suppress the rule for naming properties in the project for tests, because it is useful to get data for the test from the properties of some class TestCaseSource, using the name of the test for which this data was prepared

Reply:
Could you please clarify which rule you are talking about?

Reply:
I can’t find it, but I thought it was. If not, why did you rename the properties? Their names are now confusing

Reply:
Most likely Resharper's error. Should be fixed now